diff --git a/src/future/http/HTTPSFuture.php b/src/future/http/HTTPSFuture.php
index 289f412..61dd8d3 100644
--- a/src/future/http/HTTPSFuture.php
+++ b/src/future/http/HTTPSFuture.php
@@ -350,9 +350,11 @@ final class HTTPSFuture extends BaseHTTPFuture {
       $domain = id(new PhutilURI($uri))->getDomain();
       if (!empty(self::$blindTrustDomains[$domain])) {
         // Disable peer verification for domains that we blindly trust.
-        curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, false);
+          curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, false);
+          curl_setopt($curl, CURLOPT_SSL_VERIFYHOST, false);
       } else {
-        curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, true);
+          curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, true);
+          curl_setopt($curl, CURLOPT_SSL_VERIFYHOST, true);
       }
 
       curl_setopt($curl, CURLOPT_SSLVERSION, 0);
